### PR TITLE
My first poem

### PR DESCRIPTION
**Pull request by jjmoonisboss**

### CODE REVIEW

File: 1st poem [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Roses are red,
+Violets are blue,
+This poem is random
```
Comment (jkschulz) [inline 1st poem]: Can you add a comma at the end of this line for consistency?

**Review by jkschulz — CHANGES_REQUESTED** [review]

File: 1st poem (hunk 1 above)
Comment (jjmoonisboss) [reply 1st poem]: > Can you add a comma at the end of this line for consistency?

Sure!

Comment (jkschulz) [reply 1st poem]: Thanks! Respond to this comment once you've made the commit, please!

Comment (jkschulz) [reply 1st poem]: Ah! I was too slow! Looks great!

**Review by jkschulz — APPROVED** [review]

**Pull request merged**